kernel-modules/support-packages/04-create-support-packages.sh

```diff
@@ -58,15 +63,22 @@ for mod_ver_dir in "${MD_DIR}/module-versions"/*; do
     filename="support-pkg-${mod_ver}-$(date '+%Y%m%d%H%M%S').zip"
     latest_filename="support-pkg-${mod_ver}-latest.zip"
 
-    cp "${LICENSE_FILE}" "${probe_dir}"/LICENSE
+    if [[ -z "$(ls -A "$probe_dir")" ]]; then
+        # non-fatal here to ensure all module versions are processed
+        echo >&2 "[WARNING] no probes downloaded for ${mod_ver}"
+        touch "${package_out_dir}/empty"
```

Comment:
I think we can simply `continue` here and drop the else part of the block

Reply:
We can also simplify logic on the GHA job a bit more if we do something like `echo "${mod_ver}" >> "${OUT_DIR}/empty"`. Then we can directly look for the file and we'll have the list of empty module versions by cat'ing the contents of the file.